src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Tables/Instance.sql

```diff
@@ -20,6 +20,7 @@ CREATE TABLE dbo.Instance (
     PartitionKey            INT                        NOT NULL DEFAULT 1,  --FK
     --instance metadata
     TransferSyntaxUid       VARCHAR(64)                NULL,
+    HasFrameMetadata        BIT                        NOT NULL DEFAULT 0
```

Comment:
would it help putting this column as include column in `IX_Instance_StudyInstanceUid_Status_PartitionKey`

Reply:
There was not a lot of perf gain